app/models/audit/event/whoami.rb

```diff
@@ -0,0 +1,80 @@
+module Audit
+  module Event
+    class Whoami
+
+      #ACCOUNT_PLACEHOLDER = "ACCOUNT_MISSING"
+      #USERNAME_PLACEHOLDER = "USERNAME_MISSING"
+
+      # Note: "username" may refer to a user or host.
+      def initialize(
+        client_ip:,
+        role_id:,
+        success:
+      )
+        @client_ip = client_ip
+        @role_id = role_id
```

Comment:
Given its content, maybe consider naming the variable `role` instead of `role_id`

Reply:
done